libs/core/src/lib/avatar-group/avatar-group.component.scss

```diff
@@ -1,769 +1 @@
-    background-color: var(--sapAccentColor2,#d04343);
-    border-color: #d04343;
-    border-color: var(--sapAccentColor2,#d04343);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-3 {
-    background-color: #db1f77;
-    background-color: var(--sapAccentColor3,#db1f77);
-    border-color: #db1f77;
-    border-color: var(--sapAccentColor3,#db1f77);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-4 {
-    background-color: #c0399f;
-    background-color: var(--sapAccentColor4,#c0399f);
-    border-color: #c0399f;
-    border-color: var(--sapAccentColor4,#c0399f);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-5 {
-    background-color: #6367de;
-    background-color: var(--sapAccentColor5,#6367de);
-    border-color: #6367de;
-    border-color: var(--sapAccentColor5,#6367de);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-6 {
-    background-color: #286eb4;
-    background-color: var(--sapAccentColor6,#286eb4);
-    border-color: #286eb4;
-    border-color: var(--sapAccentColor6,#286eb4);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-7 {
-    background-color: #0f828f;
-    background-color: var(--sapAccentColor7,#0f828f);
-    border-color: #0f828f;
-    border-color: var(--sapAccentColor7,#0f828f);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-8 {
-    background-color: #7ca10c;
-    background-color: var(--sapAccentColor8,#7ca10c);
-    border-color: #7ca10c;
-    border-color: var(--sapAccentColor8,#7ca10c);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-9 {
-    background-color: #925ace;
-    background-color: var(--sapAccentColor9,#925ace);
-    border-color: #925ace;
-    border-color: var(--sapAccentColor9,#925ace);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
-.fd-avatar-group .fd-avatar-group__more-button--accent-color-10 {
-    background-color: #647987;
-    background-color: var(--sapAccentColor10,#647987);
-    border-color: #647987;
-    border-color: var(--sapAccentColor10,#647987);
-    color: #fff;
-    color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
-}
+@import '~fundamental-styles/dist/avatar-group';
```

Comment:
`0.16.1` version doesn't bring valid styles. Let's for `0.17.0` styles migration
![image](https://user-images.githubusercontent.com/26483208/112823395-828b0400-9089-11eb-998d-9345ffeecef0.png)